match/spec/runner_spec.rb

```diff
@@ -178,6 +181,8 @@
           expect(spaceship).to receive(:certificates_exists).and_return(true)
           expect(spaceship).to receive(:profile_exists).and_return(true)
           expect(spaceship).to receive(:bundle_identifier_exists).and_return(true)
+          expect(Match::Utils).to receive(:get_cert_info)
```

Comment:
I think you don’t need this line

Reply:
Uhh been a while since I created this PR so I'm don't remember 100%, but I think I do need it - iirc this flow calls `get_cert_info` twice. plus wouldn't the tests be failing if there were an extraneous `expect`?

Reply:
yea, you're right, for some reason i was reading it like it was `allow` and not `expect` 👍